### PR TITLE
Gh 45

### LINKED ISSUE #45: IPPrefixFIeld
Add `IPPrefixField` to `schema.inet`

### LINKED ISSUE #46: fix abc deprecation warning
```confu/schema/core.py:592: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.10 it will stop working```

### PR DESCRIPTION
**Pull request by djeromov**
fixes #45 
fixes #46

### CODE REVIEW

**Comment by djeromov** [comment]
this branches off of `updates` branch and includes latest revisions that fixed #46 fix abc deprecation warning

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,4 @@ release.sh
 .tox
 coverage.xml
 /dist
+/.vscode
```
Comment (grizz) [inline .gitignore:11]: This should be put it in `~/.gitignore` since not everyone uses vscode

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (grizz) [inline .vscode/settings.json:1]: This should not be included at all

File: tests/test_validation.py [hunk 3]
```diff
@@ -106,6 +109,11 @@
         (IpAddress, ipv4, ipaddress.IPv4Address(ipv4), ipv6, {"protocol": 4}),
         (IpAddress, ipv6, ipaddress.IPv6Address(ipv6), ipv4, {"protocol": 6}),
         (IpAddress, "", "", "1.2.3", {"blank": True}),
+        (IpNetwork, ipv4_n, ipaddress.IPv4Network(ipv4_n), "1.2.3/24", {}),
+        (IpNetwork, ipv6_n, ipaddress.IPv6Network(ipv6_n), "1.2.3/64", {}),
+        (IpNetwork, ipv4_n, ipaddress.IPv4Network(ipv4_n), ipv6_n, {"protocol": 4}),
+        (IpNetwork, ipv6_n, ipaddress.IPv6Network(ipv6_n), ipv4_n, {"protocol": 6}),
+        # (IpNetwork, "", "", "10.10.0.0/24", {"blank": True}),
```
Comment (vegu) [inline tests/test_validation.py:116]: This is testing that a blank value initialization works and should not be uncommented

Comment (djeromov) [reply tests/test_validation.py]: I have fixed this by changing the value_fail (4th param) to "10.10.0/24"

**Review by vegu — APPROVED** [review]

**Pull request merged**